proto/ethermint/evm/v1/evm.proto

```diff
@@ -18,9 +18,12 @@ message Params {
   repeated int64 extra_eips = 4 [(gogoproto.customname) = "ExtraEIPs", (gogoproto.moretags) = "yaml:\"extra_eips\""];
   // chain_config defines the EVM chain configuration parameters
   ChainConfig chain_config = 5 [(gogoproto.moretags) = "yaml:\"chain_config\"", (gogoproto.nullable) = false];
+  // list of allowed eip712 msgs and their types
+  repeated EIP712AllowedMsg eip712_allowed_msgs = 6
```

Comment:
Field 6 is previously `eip712_allowed_msgs` on our fork + mainnet, so to be backwards compatible the new upstream `allow_unprotected_txs` field is updated to field 7.

Breaking proto CI job considers it breaking since upstream has it field 6.

Reply:
Hmm I think we will have to keep the 6 for our field, and ignore the breaking proto OR update it to check against our previous version.  Downside here is that other codecs using ethermint proto's from upstream won't be compatible. There are some other options like migrating to our own proto type or changing the namespace, worth a discussion.

Reply:
Another option could be to bite the bullet on the breaking codec change and separate upstream param's from our custom parameters.